File: BOJ/10000 ~ 11000/P10813.java
// https://www.acmicpc.net/problem/10813
import java.io.*;
import java.util.*;

public class P10813 {
    public static void main(String[] args) throws IOException {
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);
        StringTokenizer st = new StringTokenizer(br.readLine());

        int n = Integer.parseInt(st.nextToken());
        int m = Integer.parseInt(st.nextToken());
        int[] basket = new int[n+1];

        // init basket
        for (int i=1; i<=n; i++) {
            basket[i] = i;
        }

        // swap
        for (int i=0; i<m; i++) {
            st = new StringTokenizer(br.readLine());
            int iBasket = Integer.parseInt(st.nextToken());
            int jBasket = Integer.parseInt(st.nextToken());

            int temp = basket[iBasket];
            basket[iBasket] = basket[jBasket];
            basket[jBasket] = temp;
        }

        for (int i=1; i<=n; i++) {
            if (i != n) {
                System.out.print(basket[i] + " ");
            } else {
                System.out.print(basket[i]);
            }
        }
    }
}

File: BOJ/10000 ~ 11000/P10813.kt
// https://www.acmicpc.net/problem/10813

import java.io.*

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))

    val (n, m) = br.readLine().split(" ").map { it.toInt() }
    val arr: IntArray = (0..n).toList().toIntArray()

    (1..m).forEach { _ ->
        val (i, j) = br.readLine().split(" ").map { it.toInt() }
        val temp = arr[i]
        arr[i] = arr[j]
        arr[j] = temp
    }

    // 0을 제외하고 나머지
    arr.slice(1..n).mapIndexed { index, i ->
        if((index + 1) != n) print("$i ")
        else print("$i")}
}
